### DISCUSSION TITLE
<p-divider> with badge: space coloring

### DISCUSSION THREAD

**Discussion by println4debug** [primefaces/community#680] [PrimeNG]
I've a p-divider-center in a card with a grey background, the spacing between the divider and the badge is white, but I want to make it grey aswell. I already tried to give the divider or the span a custom id and then set the bg color. But this has no changes to it. Does anyone know how I can change it? 
`        <p-divider-center >
          <span class="p-tag">Info</span>
        </p-divider-center>`

**Comment by println4debug** [comment]
I want to add that i already found out, that <p-divider-content> in the themes.css seems to change that attribute but there is no way for me to change that in my components css, even with the !important tag. `#data .p-divider .p-divider-content {
  background-color: grey !important;
}`

the only solution ive found was to change it globally in the global .css but this seems weired to me that i cant change it in the components css

**Comment by cetincakiroglu** [comment]
Hi,

could you please share a stackblitz reproducer so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
